packages/gatsby-theme-sso-ui-kit/gatsby-browser.js

```diff
@@ -1,9 +1,8 @@
+import { navigate } from 'gatsby';
 import { Auth0Provider } from '@auth0/auth0-react';
 import ConfigContext from './src/components/config-context';
 
-const onRedirectCallback = (appState) => {
-  window.location.replace(appState.returnTo);
-};
+const onRedirectCallback = (appState) => navigate(appState?.returnTo || '/');
```

Comment:
Does using gatsby navigate fix some other known bug or did you apply this just generally for being a best practice?

Reply:
Best practice. I found this pattern used in the library example. But it might be the cause of what you've seen above. I'll have a better look into it

Reply:
@nkuehn It turns out that the issue was indeed caused by this change. Would you mind give it another go? It works for me but I'd like to double check